### DISCUSSION TITLE
Forge support

### DISCUSSION THREAD

**Discussion by aws0jam** [plan-player-analytics/Plan#3209] [Q&A]
Hello! Will there be forge support in the future in addition to fabrick?

**Comment by AuroraLS3** [comment]
I don't know. If someone gets the build system for Forge to work, sure. I haven't gotten it to work.

**Comment by boldymann** [comment]
Hello, still not support for forge servers?
Yes i know threat is almost a jear old, but still it would be perfect for lots of admins
